Jenkinsfile

```diff
@@ -18,6 +18,24 @@ pipeline {
     parameters {
         string(name: 'PAVICS_HOST', defaultValue: default_pavics_host,
                description: 'PAVICS host to run notebooks against.', trim: true)
+        // TEST_MAGPIE_AUTH enables the evaluation of end-2-end access to some secured resources by Twitcher/Magpie
+        // and validate that resulting resources are found (or blocked) after proxy resolution of granted/denied access.
+        // If Authentication/Authorization is not needed or not employed for your instance, this should be disabled to
+        // avoid failures.
+        // NOTE:
+        //   This test suite might require manual clean-up on failure (if critical error).
+        //   The script attempts to remove everything, but could be incapable of doing so if Magpie become inaccessible
+        //   midway during execution. Employed users/groups are not critical (not modifying existing users permissions),
+        //   but could 'pollute' the user list over time.
+        booleanParam(name: 'TEST_MAGPIE_AUTH', defaultValue: true,
+                     description: 'Check the box to test Authentication/Authorization using Magpie/Twitcher services.')
+        // below credentials are the defaults from bootstrap script:
+        //   https://github.com/bird-house/birdhouse-deploy/blob/master/birdhouse/scripts/create-magpie-authtest-user
+        // overrides are needed when using custom values (strongly recommended for production server with real data)
+        password(name: 'TEST_MAGPIE_ADMIN_USERNAME', defaultValue: 'admin',
```

Comment:
This is just a username, should it be just `string` instead of `password`?

Reply:
`Password` masks the value, which is better IMHO.

Reply:
Any particular reason to mask the username?  Usually that do not have to be masked.  Only the password need to be masked.

Anyhow, I do not feel strongly about this so if you want to keep it this way, I have no further comment, I was just curious.

Reply:
technically speaking it is half the credentials, but yeah, not super critical